src/main/java/rasputin/command/InvalidCommand.java

```diff
@@ -0,0 +1,33 @@
+package rasputin.command;
+
+import rasputin.gui.Ui;
+
+/**
+ * Represents an invalid command due to invalid parameters or input.
+ */
+public class InvalidCommand extends Command {
+
+    private String message;
```

Comment:
Maybe the name of the message can be more specific so readers can know what message we are referring to.